### PR TITLE
add the Achievements

### PR DESCRIPTION
**Pull request by karthik193**
Please Check the Vertical time line and the colors too

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +18,8 @@
     "prop-types": "15.7.2",
     "react": "16.12.0",
     "react-dom": "16.12.0",
-    "react-helmet": "6.1.0"
+    "react-helmet": "6.1.0",
+    "react-vertical-timeline-component": "^3.0.2"
   },
```
Comment (abhishekthalatoty) [inline package.json:23]: Remove all "^"

File: src/pages/components/Achievements.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        },
+        {
+            title:"ACM ICPC 2019",
+            tagline:"tagline...",
+            description:"Description...",
+            date:"20 Jul 2020",  
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        }
+    ]
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:33]: Isolate all of the Data and put them in a JSON file.

File: src/pages/components/Achievements.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+export default function Achievements()
+{
+    
+    let  TuringHutAchievements  = [
+        {
+            title:"Title",
+            tagline:"tagline...",
+            description:"Description...",
+            date:"20 Jul 2020",  
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:17]: I would suggest to rename `imageAddress` to `imageUrl`

File: src/pages/components/Achievements.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            title:"ACM ICPC 2019",
+            tagline:"tagline...",
+            description:"Description...",
+            date:"20 Jul 2020",  
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        }
+    ]
+    return(
+        <VerticalTimeline animate={true} >
+            {TuringHutAchievements.map((eachAchievement)=>{
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:36]: I would suggest to replace `eachAchievement` with `achivement`

File: src/pages/components/Achievements.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                        style={{maxWidth:"160px", margin:"10px" , minHeight:"200px"}} ></img>
+                    </Grid>
+                    <Grid xs={12} md={7} container item justify="flex-start"  style={{backgroundColor:"" , marginBottom:"10px", color:"#fff" }}> 
+                          <ul style={{listStyle:"none" , padding:"0px" }}>
+                           <li><h3 className="vertical-timeline-element-title">{eachAchievement.title}</h3></li>
+                           <li><h5 className="vertical-timeline-element-subtitle">{eachAchievement.tagline}</h5></li> 
+                           <li><p>{eachAchievement.description}</p></li>
+                          </ul>                      
+                            
+                    </Grid>
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:65]: Use `Typography` whenever using Headings or Paragraphs

File: src/pages/index.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import React from 'react'
+import Achievements from './components/Achievements'
 
 const IndexPage = () => <h1>Hello world!</h1>
-
-export default IndexPage
+const Achievement = ()=> <Achievements />
+export default Achievement
```
Comment (abhishekthalatoty) [inline src/pages/index.js:6]: Add Achivements tag inside IndexPage and export it by default

File: src/pages/components/Achievements.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+                </Grid>
+                
+               
+            </VerticalTimelineElement>
+                ) ;
+            })}
+        </VerticalTimeline>
+    )
+}
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:74]: Prettier is not working, Please look into it.

File: src/pages/components/Achievements.js [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+                icon = {<icons.Image />}
+                
+                > 
+                <Grid
+                container
+                direction="row"
+                justify="space-evenly"
+                spacing= {3}
+                >
+                    <Grid xs={12} md={5} justify="flex-start" alignItems="center" container item style={{backgroundColor:""}} >
```
Comment (abhishekthalatoty) [inline src/pages/components/Achievements.js:53]: What is the backgroundColor?

**Comment by abhishekthalatoty** [comment]
Branch Name should Specify the new Functionality/Feature

File: package.json (hunk 1 above)
Comment (karthik193) [reply package.json:23]: Yeah okay

File: src/pages/components/Achievements.js (hunk 3 above)
Comment (karthik193) [reply src/pages/components/Achievements.js:17]: Okay

File: src/pages/components/Achievements.js (hunk 2 above)
Comment (karthik193) [reply src/pages/components/Achievements.js:33]: Okay,

File: src/pages/components/Achievements.js (hunk 5 above)
Comment (karthik193) [reply src/pages/components/Achievements.js:65]: Okay

File: src/pages/index.js (hunk 6 above)
Comment (karthik193) [reply src/pages/index.js:6]: Okay

File: src/pages/components/Achievements.js (hunk 8 above)
Comment (karthik193) [reply src/pages/components/Achievements.js:53]: Oh! I was using it for styling the Grid.

File: src/pages/components/Achievements.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+                    {/* Random Image */}
+                    <img src= {eachAchievement.imageAddress}
+                        style={{maxWidth:"160px", margin:"10px" , minHeight:"200px"}} ></img>
+                    </Grid>
+                    <Grid xs={12} md={7} container item justify="flex-start"  style={{backgroundColor:"" , marginBottom:"10px", color:"#fff" }}> 
+                          <ul style={{listStyle:"none" , padding:"0px" }}>
```
Comment (sindhujaponnuru) [inline src/pages/components/Achievements.js:59]: I would suggest putting all the styling css in a different file and importing changes. Inline styling is only suggested for 1 or 2 attributes.

Comment (karthik193) [reply src/pages/components/Achievements.js:59]: Okay

**Pull request closed without merging**